.travis.yml

```diff
@@ -0,0 +1,42 @@
+# This config uses industrial_ci (https://github.com/ros-industrial/industrial_ci.git).
+# For troubleshooting, see readme (https://github.com/ros-industrial/industrial_ci/blob/master/README.rst)
+
+language: cpp # optional, just removes the language badge
+compiler:
+  - gcc
+os:
+  - linux
+branches:
+  only:
+  - master
+
```

Comment:
Consider adding a regex for branches feature/*, so they can be checked before submitting a pull request